integration-tests/bats/sql-fetch.bats

```diff
@@ -217,7 +217,7 @@ teardown() {
 @test "sql-fetch: dolt_fetch unknown remote fails" {
     cd repo2
     dolt remote remove origin
-    run dolt sql -q "select dolt_fetch('unknown', 'master')"
+    run dolt sql -q "select dolt_fetch('unknown')"
```

Comment:
Is this not a valid test?

Reply:
both are valid, but remotes are resolved before refspecs, so i thought the addition of a branch name distracted from what was actually being tested